changelogs/DP-22028.yml

```
@@ -0,0 +1,6 @@
+Changed:
+  - project: Patternlab
+    component: Leaftlet Map
```

```suggestion
    component: LeaftletMap
```
Please use PascalCase